pkg/nvidia/Dockerfile

```diff
@@ -0,0 +1,102 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM lfedge/eve-alpine:82df60e43ab9f8c935584b8c7b4d0a4b0271d608 as build-base
+ENV BUILD_PKGS="autoconf automake build-base coreutils gettext git glib-dev go libtool libmd-dev ncurses-dev tar xz-dev yq zstd-dev"
+
+RUN eve-alpine-deploy.sh
+
+# This is a arm64 (only) package, nothing to do for other architectures
+FROM build-base as build-amd64
+FROM build-base as build-riscv64
+
+FROM build-base as build-arm64
+
+# Setup git to apply patches
+RUN git config --global user.email 'builder@projecteve.dev' && \
+    git config --global user.name 'Project EVE'
+
+# Jetson Linux tarball
+ENV JETSON_LINUX=https://developer.nvidia.com/downloads/embedded/l4t/r35_release_v5.0/release/jetson_linux_r35.5.0_aarch64.tbz2
+
+ADD ${JETSON_LINUX} /jetson_linux.tbz2
+
+# We need to build dpkg in order to get zstd compression support
```

Comment:
Two questions:

1. I didn't understand this. Why do we need dpkg to get zstd compression support?
2. Why don't we just use the [alpine dpkg package](https://pkgs.alpinelinux.org/package/edge/main/aarch64/dpkg) (that sounds funny when you say it out loud)?

Or are you saying not that we need dpkg in order to use zstd on other things, but if a `.deb` package is compressed with zstd, then we need _dpkg itself_ to support zstd, and the version packaged with alpine does not? And so we need to compile it?

Reply:
Ah, and Alpine 3.20 has support for it, hence your comment below?

Reply:
Can we put that in the comment here, so it is clear? That the `.deb` we will unpack might have zstd compression support, and our Alpine package dpkg is from alpine 3.16 and does not support it, and when we switch to alpine 3.20 we can get rid of all of this? And maybe a TODO line as well?